src/java/boa/functions/BoaAstIntrinsics.java

```diff
@@ -209,9 +210,12 @@ private static void openMap() {
 		final Configuration conf = new Configuration();
 		try {
 			final FileSystem fs = FileSystem.get(conf);
-			final Path p = new Path("hdfs://boa-njt/",
```

Comment:
Could you please send the location (file, function) and the intended change, so that the change can be made as part of this pull request?